include/small/abstract_layer.hpp

```diff
@@ -270,7 +276,7 @@ void inline kernel(
         if (op_type == OP_MAX_POOL || op_type == OP_MUL)
         {
             /// @note using platform C_ob
```

Comment:
With the change from FLOAT_C_ob to _C_ob, this comment is no longer correct. Remove?